src/pages/WorkspacesList/index.tsx

```diff
@@ -150,7 +150,7 @@ export default class WorkspacesList extends React.PureComponent<Props, State> {
 
     return [
       {
-        title: 'Open in Verbose Mode',
+        title: 'Open in Verbose mode',
```

Comment:
Seems it still differs from docs: 

> . When it fails to start, click on the displayed *Open in Verbose Mode* link.

https://github.com/eclipse/che-docs/pull/1972/files#diff-4f8270529c618f1a4a7763f11bf6c4e969bb841cf4375ce09fe8ba27bff08bf5R28

Or it's another place?

Reply:
Thank you for pointing that out, it was my mistake. This PR [Fix inconsistent capitalization #1974](https://github.com/eclipse/che-docs/pull/1974) fixes it.